completions/ip

```diff
@@ -8,11 +8,21 @@ _comp_cmd_ip__iproute2_etc()
 
 _comp_cmd_ip__netns()
 {
-    _comp_compgen_split -- "$(
+    local unquoted
+    _comp_split -l unquoted "$(
         {
             ${1-ip} -c=never netns list 2>/dev/null || ${1-ip} netns list
-        } | _comp_awk '{print $1}'
+        } | command sed -e 's/ (.*//g'
```

Comment:
```suggestion
        } | command sed -e 's/ (.*//'
```

I think it matches only up to once because  `.*` anyway matches to the end of the line.

Reply:
Yeah, it works without, removed